src/textfile_type.h

```diff
@@ -18,7 +18,9 @@ enum TextfileType {
 	TFT_CHANGELOG,          ///< NewGRF changelog
 	TFT_LICENSE,            ///< NewGRF license
 
-	TFT_END,
+	TFT_END, // This marker is used to generate the above three buttons in sequence by various of places in the code.
```

Comment:
Having end not meaning end might trip someone up in the future. Maybe introduce `TFT_CONTENT_BEGIN`/`TFT_CONTENT_END` for the current iterations, that are clearly not meant to be used for all files?